Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt
+++ CMakeLists.txt	2018-02-21 23:16:30.740788632 +0100
@@ -5,7 +5,7 @@
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_STANDARD 14)
 
-set(REQUIRED_QT_VERSION "5.9.0")
+set(REQUIRED_QT_VERSION "5.8.0")
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Core Network Qml Quick Test Sql Multimedia Svg Gui Widgets QuickTest)
 
 set(REQUIRED_KF5_VERSION "5.32.0")
